app/src/styles.sass

```diff
@@ -44,6 +44,13 @@ input
       background-color: tame($userbuttonbgcolor)
       color: tame($useboxfgcolor)
 
+.formactions.login
+  justify-content: center
+  height: 52px
+  padding-top: 14.5px
+  padding-bottom: 14.5px
```

Comment:
I don't agree with this additional space. In the original code, the space between the bottom of the button and the bottom border of the form is roughly the same as the space between the top border of the form and the fist label. This is not the case any more with this additional padding.

Reply:
ok I will delate the bottom padding, but it seems a little strange with just the margin above. I added the padding just to make the button appear on the center of the form(same space up and down). Okay I will delate it for the consistency.

Reply:
Ok, I understand why we disagree. You're considering the form when it's empty, whereas I'm considering it when it's filled. I'm uncertain which solution is best now...